### PR TITLE
Update SirCAL logging

### PR DESCRIPTION
**Pull request by benjaminysmith**
### Description
Update SirCAL logging to support replicating alerts through elastic

### Changelog
- Log aggregated SLA violations
- Change some messages we don't need in the logs back to prints

### Fixes 
- Tweaks #612

### CODE REVIEW

File: sir_complainsalot/delphi_sir_complainsalot/run.py [hunk 1]
```diff
@@ -37,14 +37,6 @@ def run_module():
                                        params["sources"], params.get("grace", 0), LOGGER))
 
     if len(complaints) > 0:
-        for complaint in complaints:
-            LOGGER.critical(event="signal out of SLA",
```
Comment (sgsmob) [inline sir_complainsalot/delphi_sir_complainsalot/run.py:41]: Why does this need to be moved?

File: sir_complainsalot/delphi_sir_complainsalot/run.py [hunk 2]
```diff
@@ -37,14 +37,6 @@ def run_module():
...
-                            message=complaint.message,
-                            data_source=complaint.data_source,
-                            signal=complaint.signal,
-                            geo_types=complaint.geo_types,
-                            last_updated=complaint.last_updated.strftime("%Y-%m-%d"))
```
Comment (sgsmob) [inline sir_complainsalot/delphi_sir_complainsalot/run.py:46]: This argument is dropped below.  Was that intentional?

**Review by sgsmob — COMMENTED** [review]
Just a few questions about what is going on here but otherwise looks good

File: sir_complainsalot/delphi_sir_complainsalot/run.py (hunk 1 above)
Comment (benjaminysmith) [reply sir_complainsalot/delphi_sir_complainsalot/run.py:41]: Old: each line-level complaint was being logged one at a time (one line per type/source/signal/geo combo)
New: only the aggregated complaint is logged (one line per type/source; signal/geo aggregated)

(this makes it much more useful for alerting)

File: sir_complainsalot/delphi_sir_complainsalot/run.py (hunk 2 above)
Comment (benjaminysmith) [reply sir_complainsalot/delphi_sir_complainsalot/run.py:46]: Yeah, in the new code it's been pulled directly into the aggregated message.

**Review by sgsmob — APPROVED** [review]

**Pull request merged**